Traceback (most recent call last):
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\jupyter_core\utils\__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "C:\Users\tbeucler\.conda\envs\JB\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\tbeucler\.conda\envs\JB\lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
#@title  Run this cell for Python library imports. Double click for the source code
import cartopy
import cartopy.feature as cfeature
import cartopy.crs as ccrs
import matplotlib as mpl
import matplotlib.pyplot as plt
from matplotlib.ticker import MaxNLocator
import numpy as np
import scipy.integrate as sin
import tensorflow as tf
from tensorflow import keras
from keras.layers import *

import h5py
import pickle
import pooch
import xarray as xr
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [1;32mIn[2], line 2[0m
[0;32m      1[0m [38;5;66;03m#@title  Run this cell for Python library imports. Double click for the source code[39;00m
[1;32m----> 2[0m [38;5;28;01mimport[39;00m [38;5;21;01mcartopy[39;00m
[0;32m      3[0m [38;5;28;01mimport[39;00m [38;5;21;01mcartopy[39;00m[38;5;21;01m.[39;00m[38;5;21;01mfeature[39;00m [38;5;28;01mas[39;00m [38;5;21;01mcfeature[39;00m
[0;32m      4[0m [38;5;28;01mimport[39;00m [38;5;21;01mcartopy[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcrs[39;00m [38;5;28;01mas[39;00m [38;5;21;01mccrs[39;00m

[1;31mModuleNotFoundError[0m: No module named 'cartopy'

